Core/src/main/java/com/paypal/android/core/analytics/AnalyticsClient.kt

```diff
@@ -0,0 +1,31 @@
+package com.paypal.android.core.analytics
+
+import android.util.Log
+import com.paypal.android.core.Http
+import com.paypal.android.core.HttpRequestFactory
+import java.util.UUID
+
+internal class AnalyticsClient(
```

Comment:
q: is the goal to instantiate a new `AnalyticsClient` for each feature client?

Reply:
Similar question [here](https://github.com/paypal/Android-SDK/pull/107#discussion_r1039909042)